flask/src/main.py

```diff
@@ -75,7 +75,6 @@ def __init__(self, import_name, *args, **kwargs):
             traces_sample_rate=1.0,
             before_send=before_send,
             traces_sampler=traces_sampler,
-            enable_db_query_source=True,
```

Comment:
is this correct?

Reply:
yep. it's defaulted on (:
![image](https://github.com/sentry-demos/empower/assets/314061/86ad4b23-31df-4cc0-8cb8-3d20f6d9cf65)
https://github.com/getsentry/sentry-python/releases/tag/1.40.0

Reply:
ah gotcha, docs didn't get updated it looks